src/universal/components/QueryRenderer/QueryRenderer.js

```diff
@@ -117,9 +117,10 @@ export default class QueryRenderer extends React.Component {
   }
 
   _requestRelease() {
```

Comment:
I was wondering this as well in the `projects-to-relay` PR - why aren't we just using the `QueryRenderer` packaged with `react-relay`?

Reply:
let's chat it up! all about subscriptions